graylog2-server/src/main/java/org/graylog/security/certutil/ca/CACreator.java

```diff
@@ -42,7 +45,7 @@ public KeyStore createCA(final char[] password,
                             .validity(certificateValidity)
             );
 
-            KeyStore caKeystore = KeyStore.getInstance("PKCS12");
+            KeyStore caKeystore = KeyStore.getInstance(PKCS12);
             caKeystore.load(null, null);
```

Comment:
I am not sure if password should not be used here as well... We can discuss that if you want.

Reply:
worked so far without setting a password on the keystore - it's written later with a password into Mongo, so I guess it's fine but I put a todo into the code.